### PR TITLE
start load test work

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

Create a feature where a platform admin can perform on load test on the development, staging, or demo tier.

For now, assume that a 'Test service' exists and that it contains an 'Example text message template', and then send 250 successful messages to the AWS simulated number for success, and 250 'failed' messages to the AWS simulated number for failures.

## Security Considerations

N/A

### CODE REVIEW

File: app/main/views/platform_admin.py [hunk 1]
```diff
@@ -771,3 +781,76 @@ def _get_user_row(r):
...
+    methods=["POST", "GET"],
+)
+@user_is_platform_admin
+def load_test():
+    """
+    The load test assumes that a service called 'Test service' exists.  It will make
+    the platform admin a member of this service if the platform is not already. All
+    messagese will be sent in this service.
+    """
+    # SIMULATED_SMS_NUMBERS = ("+14254147755", "+14254147167")
```
Comment (ccostino) [inline app/main/views/platform_admin.py:797]: We have these numbers defined as a global config for the main Flask app too, right?

File: app/main/views/platform_admin.py [hunk 2]
```diff
@@ -771,3 +781,76 @@ def _get_user_row(r):
...
+        }
+        _send_notification(service["id"], example_template["id"])
+    for _ in range(0, 3):
+        session["recipient"] = "+14254147167"
+        session["placeholders"] = {
+            "day of week": "Monday",
+            "color": "blue",
+            "phone number": "+14254147167",
+        }
+        _send_notification(service["id"], example_template["id"])
```
Comment (ccostino) [inline app/main/views/platform_admin.py:817]: I'd add a couple comments to each `for` block just to note which is the success and which is the failure - or, we can swap the numbers out for the config variables and/or constants/enums that are clearly labeled too.  Food for future thought!

**Review by ccostino — COMMENTED** [review]
Just a couple of notes around the simulated numbers, but there are no major concerns from me in the approach.  Thanks, @terrazoon!

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

(And I realized that the app config with the simulated numbers as on the API side, not yet on the admin side!)

**Review by A-Shumway42 — APPROVED** [review]
LGTM. No concerns here.

**Pull request merged**